lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -0,0 +1,114 @@
+var edx = edx || {};
+
+(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortDiscussionsView = Backbone.View.extend({
+
+        /**
+         Add/Remove the disabled attribute on given element.
+
+         Args:
+            $element (JQuery element): The element to disable/enable.
+            disable (Bool): The flag to add/remove 'disabled' attribute.
+        **/
+        setDisabled: function($element, disable) {
+            if (disable) {
```

Comment:
This would be easier to read as:

    $element.prop('disabled', disable ? 'disabled' : false);